Dockerfile

```diff
@@ -16,18 +16,20 @@ RUN apt-get update && apt-get install -y \
     libncurses5 libncurses5-dev \
     tcl environment-modules \
     && apt-get clean autoclean
+
+# install compiler
+RUN apt-get install -y build-essential
+
 # install get_tools.py script
 #   requires connection to XMOS network at build and run time
 #   if not possible, find another way to install the tools
-RUN mkdir -m 777 /XMOS && cd /XMOS \
-    && wget -q https://github0.xmos.com/raw/xmos-int/get_tools/master/get_tools.py \
-    && chmod a+x get_tools.py \
+RUN mkdir -m 777 /XMOS
+ADD https://github0.xmos.com/raw/xmos-int/get_tools/master/get_tools.py /XMOS/
+RUN cd /XMOS \
+    && chmod 755 get_tools.py \
     && echo "export MODULES_SILENT_SHELL_DEBUG=1\nexport MODULEPATH=/XMOS/modulefiles:/XMOS/template_modulefiles\nexport PATH=$PATH:/XMOS" \
```

Comment:
The `MODULES_SILENT_SHELL_DEBUG=1\nexport MODULEPATH=/XMOS/modulefiles:/XMOS/template_modulefiles\nexport` bits should no longer be needed.  

`"export PATH=$PATH:/XMOS"` should be all that is needed.